maps-app-ios.xcodeproj/project.pbxproj

```diff
@@ -741,6 +756,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
`English` is a deprecated identifier, so I would suggest changing `developmentRegion` to `en` instead.

Reply:
Interesting. Is that a relatively recent deprecation?

Reply:
@nixta I'm not sure, but the warning about it being deprecated appeared in Xcode 10.2.